src/main/java/com/milaboratory/mixcr/export/AirrColumns.java

```diff
@@ -80,23 +87,27 @@ public String extractValue(AirrVDJCObjectWrapper object) {
     }
 
     public static final class RevComp implements FieldExtractor<AirrVDJCObjectWrapper> {
+        @NotNull
         @Override
         public String getHeader() {
             return "rev_comp";
         }
 
+        @NotNull
         @Override
-        public String extractValue(AirrVDJCObjectWrapper object) {
+        public String extractValue(@NotNull AirrVDJCObjectWrapper object) {
             return "F";
         }
     }
 
     public static class Productive implements FieldExtractor<AirrVDJCObjectWrapper> {
+        @NotNull
```

Comment:
No way we will be able to do it everywhere 😔, let's just put ignore mark for this check for the whole file.